po-man/meson.build

```diff
@@ -0,0 +1,38 @@
+po4a = find_program('po4a', required : false)
+
+if not (asciidoctor.found() and po4a.found())
+  subdir_done()
+endif
+
+awk = find_program('awk', required : true)
+locales_output = run_command(awk, '/\[po4a_langs\]/ {for (i=2; i<=NF; i++) print $i}', 'po4a.cfg', check : true)
```

Comment:
meson depends on python anyways, so a python script could be used to avoid an awk dependency.